workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/layout-page-templates/display-page-templates/lesson/page-definition.json

```diff
@@ -200,7 +226,7 @@
 	"settings": {
 		"colorSchemeName": "01",
 		"masterPage": {
-			"key": "liferay-learn"
+			"key": "liferay-learn-refactor"
```

Comment:
Hi, @GabrielFernandoS 
Change it back to liferay-learn, which is the name of the master page.